packages/components/src/radio/src/Radio.css

```diff
@@ -49,80 +50,94 @@
 /* CONTENT | LABEL | REVERSE */
 .o-ui-radio-reverse .o-ui-radio-label {
     margin-left: 0;
-    margin-right: var(--o-ui-sp-2);
+    margin-right: var(--hop-space-inline-sm);
 }
 
 /* CONTENT | ICON */
 .o-ui-radio-icon {
-    margin-left: var(--o-ui-sp-1);
+    margin-left: var(--hop-space-inline-sm);
 }
 
 /* CONTENT | ICON | REVERSE */
 .o-ui-radio-reverse .o-ui-radio-icon {
     margin-left: 0;
-    margin-right: var(--o-ui-sp-1);
+    margin-right: var(--hop-space-inline-sm);
 }
 
 /* CONTENT | COUNTER */
 .o-ui-radio-counter {
-    margin-left: var(--o-ui-sp-2);
+    margin-left: var(--hop-space-inline-sm);
     color: var(--o-ui-text-alias-primary);
 }
 
 /* CONTENT | COUNTER | REVERSE */
 .o-ui-radio-reverse .o-ui-radio-counter {
     margin-left: 0;
-    margin-right: var(--o-ui-sp-2);
+    margin-right: var(--hop-neutral-text);
 }
 
 /* VALIDATION */
 .o-ui-radio-invalid {
-    color: var(--o-ui-text-alias-alert);
+    color: var(--hop-danger-text-weak);
 }
 
 .o-ui-radio-invalid .o-ui-radio-button {
-    border-color: var(--o-ui-b-alias-alert);
+    border-color: var(--hop-danger-border-strong);
 }
 
 .o-ui-radio-invalid.o-ui-radio:hover > input[type="radio"]:not([disabled]) + .o-ui-radio-button,
 .o-ui-radio-invalid.o-ui-radio-hover > input[type="radio"]:not([disabled]) + .o-ui-radio-button {
-    border-color: var(--o-ui-b-alias-alert-hover);
+    border-color: var(--hop-danger-border-strong);
+    background-color: var(--hop-danger-surface);
 }
 
 .o-ui-radio.o-ui-radio-invalid .o-ui-radio-button::before {
-    background-color: var(--o-ui-bg-alias-alert);
+    background-color: var(--hop-neutral-border-hover);
 }
 
 /* STATE | CHECKED */
+.o-ui-radio-checked .o-ui-radio-button {
+    background-color: var(--hop-neutral-surface-active);
+    border-color: var(--hop-neutral-border-active);
+}
+
 .o-ui-radio-checked .o-ui-radio-button::before {
-    transform: scale(0.5);
+    transform: scale(1);
 }
 
 .o-ui-radio-checked.o-ui-radio-invalid .o-ui-radio-button {
-    border-color: var(--o-ui-b-alias-alert);
+    background-color: var(--hop-danger-surface-strong);
+    border-color: var(--hop-danger-border-strong);
+}
+
+.o-ui-radio-checked.o-ui-radio:hover > input[type="radio"]:not([disabled]) + .o-ui-radio-button,
+.o-ui-radio-checked:not(.o-ui-radio-disabled).o-ui-radio-hover .o-ui-radio-button {
+    border-color: var(--hop-neutral-border-active);
```

Comment:
Active token being used on an hover state feels odd.

Reply:
this is because the hover effect is similar to active 

![Capture d’écran, le 2023-10-20 à 12 22 36](https://github.com/gsoft-inc/wl-orbiter/assets/2379339/a95c9ff2-fcee-45dc-931c-63b1d5b44cc0)

Reply:
shouldn't it be --hop-neutral-border-hover and it's what's in design.

Reply:
You may also have to set     transform-origin: top left;